### DISCUSSION TITLE
Issues about a page are linked at the bottom of the page

### DISCUSSION THREAD

**Discussion by MrBrain295** [mdn/mdn-community#114] [Platform]
I think it might be helpful to have a section at the bottom of a page linking to issues about that page.

**Comment by bershanskiy** [comment]
Just to make sure we are on the same page: this suggestion is about improving the discoverability of GitHub issues about a particular page by creating a list of related issues, correct? Effectively, this extra section would just contain a list of links to GitHub issues in `mdn/content` repo, right?

**Reply by MrBrain295** [reply]
No more of an automaticly generated part that is added when the site is being built that links to all the issues that have that page as the mdn url in the issue.
